arviz/plots/backends/matplotlib/ecdfplot.py

```diff
@@ -0,0 +1,189 @@
+"""Matplotlib ecdfplot."""
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.stats import uniform, binom
+
+from arviz.plots.plot_utils import _scale_fig_size
+from arviz.plots.backends.matplotlib import backend_kwarg_defaults, create_axes_grid, backend_show
+
+
+def plot_ecdf(
+    values,
+    values2,
+    distribution,
+    difference,
+    pit,
+    confidence_bands,
+    granularity,
+    num_trials,
+    alpha,
+    figsize,
+    ecdf_fill,
+    ax,
+    show,
+    backend_kwargs,
+):
+    """Matplotlib ecdfplot."""
+    if backend_kwargs is None:
+        backend_kwargs = {}
+
+    backend_kwargs = {
+        **backend_kwarg_defaults(),
+        **backend_kwargs,
+    }
+
+    (figsize, _, _, _, _, _) = _scale_fig_size(figsize, None)
+    backend_kwargs.setdefault("figsize", figsize)
+    backend_kwargs["squeeze"] = True
+
+    if ax is None:
+        _, ax = create_axes_grid(1, backend_kwargs=backend_kwargs)
+
+    n = len(values)
+
+    if confidence_bands:
+        if pit:
+            x = np.linspace(1 / granularity, 1, granularity)
+            z = x
+            values = (
+                distribution(values)
+                if distribution
+                else compute_ecdf(values2, values) / len(values2)
+            )
+        else:
+            x = np.linspace(values[0], values[-1], granularity)
+            z = distribution(x) if distribution else compute_ecdf(values2, x) / len(values2)
+
+        gamma = get_gamma(n, z, granularity, num_trials, alpha)
+        lower, higher = get_lims(gamma, n, z)
+
+        x_coord, y_coord = [], []
+        if not difference:
+            for _, x_i in enumerate(x):
```

Comment:
When plotting a non-difference ECDF where the first point is the minimum of the sample points, you want to add an additional point to the beginning so that the ECDF plot hits the y-axis.

Reply:
Just to be sure, here you mean that I can add a point before the minimum so that the ecdf at that point is 0?

Reply:
That's right, yes. Usually this is done by either adding `x=-Inf`, `y=0` (some step function implementations will just draw a vertical line to the `y`-axis in that case), or just duplicating the lowest `x` value and prepending `y=0`.